sound/soc/sof/sof-client-probes-ipc4.c

```diff
@@ -8,6 +8,7 @@
 #include <sound/soc.h>
 #include <sound/sof/ipc4/header.h>
 #include <uapi/sound/sof/header.h>
+#include "sof-audio.h"
```

Comment:
@jsarha btw the 1st triplet above ```16777220,0,256``` is better in HEX as ```0x1000004, 0x0, 0x100```

Reply:
@lgirdwood , the reason for the decimal format is the the fackt that its decimal numbers that are expected when defining the probe points. So if we want to be able to copy-paste the triplets to probe_points debgfs file, then we want to print decimal numbers. We can of course change the write interface too so that it expect hex-numbers or is sensitive to "0x" prefix. It would take a day or two to implement and properly test that, should I go that way?

Reply:
Pls make hex, the hex value also make sense when introspection is performed by anyone debugging as they map to bit masks within the IDs, i.e. its not a coincidence we have 0x1000004 and 0x100 as values.

Reply:
It appears that the input side already supported 0x-prefixed hex inputs. So this was a simple change after all.